roles/storage-demo-nodeconfig/tasks/deprovision.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: "Remove rule: Allow ceph OSD traffic"
```

Comment:
@rthallisey How does firewall configuration is being cared when deploying using an APB?

Reply:
This is split from the other role because it runs tasks that change the hosts.  For security reasons, APBs don't change anything on the hosts.  Changing this is on our radar though, but it needs to come after the service-catalog adds rbac for apbs so we can give apbs that have cluster-admin this level of host and cluster access.

Reply:
I am trying to use a formulaic method for all storage roles based on what is being done for the kubevirt role.  Therefore I followed the pattern of using the action parameter to select the playbook to run.  The storage-demo and storage-demo-nodeconfig roles need to be separate because they target separate hosts.  The former runs on localhost and fits nicely into an APB.  The later runs on the nodes themselves and may have issues as an APB.  We might move the nodeconfig into the openshift deployment flow and include the role only when the storage_role == 'storage-demo'.